src/style.css

```
@@ -1 +1,66 @@
-/* Styles go here */
+* {
+  list-style: none;
+  text-decoration: none;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 0 50px;
+}
+
+.header__content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+img {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+}
+
+.nav__item:not(:last-child) {
+  margin: 0 20px 0 0;
+}
+
+.nav__item a {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  position: relative;
+}
+
+.nav__link {
+  color: #000;
+}
+
+.nav__item a:hover,
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  width: 37px;
```

```suggestion
  width: 100%;
```
is-active буде застосовуватись до будь-якого активного пункту меню. Не завжди у них буде ширина 37px